storage.conf

```diff
@@ -32,6 +32,10 @@ graphroot = "/var/lib/containers/storage"
 #
 # rootless_storage_path = "$HOME/.local/share/containers/storage"
 
+# Transient store mode makes all container metadata be saved in temporary storage
+# (i.e. runroot above). This is faster, but doesn't persist across reboots.
+# transient_store = true
```

Comment:
If users are expected to manually remove data after reboot, this must be prominently documented.

Reply:
@alexlarsson can you add a note about cleaning up the storage on reboot?